doc/tinyramfs.5.scd

```diff
@@ -0,0 +1,98 @@
+tinyramfs(5) "tinyramfs" "2022-05-09"
+# NAME
+tinyramfs - configuration file
+
+# DESCRIPTION
+A tinyramfs configuration file is written in POSIX sh. Configuration is done 
+via variable asignment (eg. *option=value*). Special characters and whitespace 
+must be quoted or escaped, more info: *sh*(1). By default tinyramfs looks to 
+*/etc/tinyramfs/config* for a configuration file.
+
+# OPTIONS
+Option marked with *(bool)* can hold a value of true or false. False if the 
+variable is unset or empty, true otherwise.
+
+Some options only apply to certain hooks and have been categorized accordingly.
+
+Options which expect a device as a value can also refer to the device via 
+*UUID*, *LABEL*, and *PARTUUID* (eg. LABEL=<label>). This requires a blkid 
+utility which supports that identifier.
+
+root
```

Comment:
Add additional newline between option and description here. Do same for other options.

Reply:
Multiple newlines don't show up in the output file, so I'll use '++' so it does, as I assume you'd want.